config/default.yaml

```diff
@@ -34,6 +34,8 @@ auth:
   admins: []
   # Default session timeout (in minutes)
   sessionTimeout: 120
+  # SameSite Cookie Option
+  sameSite: false
```

Comment:
should this be the default option? we can set the default as `strict` and relax it by COOKIE_SAME_SITE.

Reply:
Done